http4s-scala-fx/src/test/scala/http4s/fx/BlazeServerFXBackendSuite.scala

```diff
@@ -0,0 +1,38 @@
+package http4s
+package fx
+
+import _root_.fx.{defaultRetryPolicy, parallel, structured, GET, POST, Structured}
+import munit.fx.ScalaFXSuite
+
+import java.net.URI
+
+class BlazeServerFXBackendSuite extends ScalaFXSuite, HttpServerFixtures:
+
+  httpServerHttp4s(pingPongApp).testFX(
+    "GET requests should be returned in non-blocking fibers") { serverResource =>
+    serverResource.use { baseServerAddress =>
+      val pongResponse: Structured ?=> (String, String, String) =
+        parallel(
+          () => URI.create(s"$baseServerAddress/ping/1").GET[String]().body,
+          () => URI.create(s"$baseServerAddress/ping/2").GET[String]().body,
+          () => URI.create(s"$baseServerAddress/ping/3").GET[String]().body
+        )
+
+      assertEqualsFX(
+        structured(pongResponse),
+        ("pong", "pong", "pong")
+      )
+    }
```

Comment:
```suggestion
    assertEqualsFX(
      structured {
        serverResource.use { baseServerAddress =>
          parallel(
            () => URI.create(s"$baseServerAddress/ping/1").GET[String]().body,
            () => URI.create(s"$baseServerAddress/ping/2").GET[String]().body,
            () => URI.create(s"$baseServerAddress/ping/3").GET[String]().body
          )
        }
      },
      ("pong", "pong", "pong")
    )
```

Since we only have a single result, we can wrap the whole thing in `assertEqualsFX`, and a structured block to provide the resource in `resource.use` a `Structured` scope to start with. `testFX` provides the `run` call and that provides a concrete `Control` instance inside the continuation, which will unwrap during test evaluation in case of failure, and the exception contained in the control token will be unwrapped and reported to the test reporter with any causes in its stack trace.

In this case, what we now get is
```scala
==> X http4s.fx.BlazeServerFXBackendSuite.GET requests should be returned in non-blocking fibers  0.099s java.lang.AssertionError: java.util.concurrent.CompletionException: java.lang.WrongThreadException: Current thread not owner or thread in flock
    at munit.fx.ScalaFXSuite.testFX$$anonfun$1(ScalaFXSuite.scala:67)
    at munit.FunFixtures.munit$FunFixtures$FunFixture$$_$test$$anonfun$2$$anonfun$2$$anonfun$2(FunFixtures.scala:34)
Caused by: java.util.concurrent.CompletionException: java.lang.WrongThreadException: Current thread not owner or thread in flock
    at java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:413)
    at java.util.concurrent.CompletableFuture.join(CompletableFuture.java:2118)
    at fx.Fiber$package$.uncancellable(Fiber.scala:23)
    at fx.instances.FxAsync.uncancelable(instances.scala:76)
    at fx.instances.FxAsync.uncancelable$$anonfun$2(instances.scala:69)
    at http4s.fx.BlazeServerFXBackend.liftedTree1$1$$anonfun$1(BlazeServerFXBackend.scala:32)
    at http4s.fx.BlazeServerFXBackend.liftedTree1$1(BlazeServerFXBackend.scala:33)
    at http4s.fx.BlazeServerFXBackend.start(BlazeServerFXBackend.scala:31)
    at http4s.fx.BlazeServerFXBackend$.apply$$anonfun$1(BlazeServerFXBackend.scala:51)
    at fx.Resource.$anonfun$5(Resource.scala:26)
    at fx.Fiber$package$.uncancellable$$anonfun$1(Fiber.scala:18)
    at java.lang.VirtualThread.run(VirtualThread.java:287)
    at java.lang.VirtualThread$VThreadContinuation.lambda$new$0(VirtualThread.java:174)
    at jdk.internal.vm.Continuation.enter0(Continuation.java:327)
    at jdk.internal.vm.Continuation.enter(Continuation.java:320)
Caused by: java.lang.WrongThreadException: Current thread not owner or thread in flock
    at jdk.internal.misc.ThreadFlock.ensureOwnerOrContainsThread(ThreadFlock.java:206)
    at jdk.internal.misc.ThreadFlock.start(ThreadFlock.java:269)
    at jdk.incubator.concurrent.StructuredTaskScope.fork(StructuredTaskScope.java:419)
    at fx.Structured$package$.forked(Structured.scala:30)
    at fx.Fiber$package$.fork(Fiber.scala:27)
    at fx.instances.FxAsync$$anon$2.apply(instances.scala:73)
    at fx.instances.FxAsync$$anon$2.apply$$anonfun$2(instances.scala:72)
    at fx.instances.FxAsync.flatMap(instances.scala:89)
    at fx.instances.FxAsync.flatMap$$anonfun$1(instances.scala:88)
    at fx.instances.FxAsync.uncancelable$$anonfun$1(instances.scala:75)
    ... 5 more
```
instead of 
```scala
==> X http4s.fx.BlazeServerFXBackendSuite.expected error running tests  0.001s fx.Continuation$ControlToken: null
```
.